src/qml/NightstandPage.qml

```diff
@@ -143,7 +140,7 @@ Item {
 
             Item {
                 width: parent.width
-                height: Dims.l(20)
+                height: rowHeight
                 opacity: nightstandEnabled.value ? 1.0 : 0.4
                 enabled: nightstandEnabled.value
                 visible: nightstandUseCustomWatchface.value
```

Comment:
Since we're already making things more consistent, I think this should be grayed out rather than disappeared when the nighstandUseCustomWatchface is untoggled

Reply:
Even added a nice animation to fade between the opacity states ;)